llvm/lib/Transforms/Yk/LivenessAnalysis.cpp

```diff
@@ -58,12 +58,10 @@ LivenessAnalysis::getSuccessorInstructions(Instruction *I) {
   return SuccInsts;
 }
 
-/// Replaces the value set behind the pointer `S` with the value set `R` and
-/// returns whether the set behind `S` changed.
-bool LivenessAnalysis::updateValueSet(std::set<Value *> *S,
-                                      const std::set<Value *> R) {
-  const bool Changed = (*S != R);
-  *S = R;
+bool LivenessAnalysis::updateValueSet(std::set<Value *> &S,
```

Comment:
I think this function still needs *a* comment.

Reply:
It's in the header.